db/institutions.yml

```diff
@@ -934,6 +934,19 @@ uhouston:
   name: University of Houston
   state: TX
   zip: fixme
+uillinoischampaign:
+  aka:
+    - uiuc
+  city: champaign
```

Comment:
we will need a ``departments:`` field.  Just add one.  Please see other entries for an example

Reply:
I made changes, please review!